### ISSUE TITLE
Building GCAM on Windows docs are outdated

### ISSUE THREAD

**Issue by rjplevin** [JGCRI/gcam-doc#66]
I'm trying to follow the docs on building GCAM on Windows, but it no longer matches the current version of Visual Studio (2017), making the instructions difficult to follow. There is no `Project -> objects-main Properties` as described, but right clicking on `objects-main` in the "Solution Explorer" pane does show the pane.

A bit more detail on how to set up the required environment vars, etc. would also be helpful. For example, after downloading a binary of boost, it's not clear what to set so VS knows about it. Also, according to one website, "only Boost 1.64 and up works with Visual Studio 2017". Good to note.

I would also like to suggest organizing the docs so the instructions for each platform are contiguous, rather than interspersed within different build steps. Currently, there's e.g., a java section, and within that, sections for Mac and Windows. Anyone trying to build the model needs to follow instructions for one platform at a time.

**Comment by rjplevin** [comment]
More info: Boost 1.64-1.66 had a "unknown compiler version" error that is gone when using v 1.69. There are lots of warnings in the GCAM code.

There are also errors in xerces:
```
2>c:\users\rjp\gcam\gcam-5.1.2-adj-coef-patch\libs\xercesc\include\xercesc\util\xerces_autoconf_config.hpp(101): error C4430: missing type specifier - int assumed. Note: C++ does not support default-int (compiling source file ..\..\containers\source\scenario.cpp)
2>c:\users\rjp\gcam\gcam-5.1.2-adj-coef-patch\libs\xercesc\include\xercesc\util\xerces_autoconf_config.hpp(101): error C2146: syntax error: missing ';' before identifier 'XMLSSize_t' (compiling source file ..\..\containers\source\scenario.cpp)
```

I haven't found any information yet on Xerces compatibility with Visual Studio 2017.

It would be great if you guys could get this working with VS 2017 and update the docs. (Sorry, Pralit!)

**Comment by rjplevin** [comment]
The Xerces-C++ lib and includes included with v5.1.2 compiled with errors in VS 2017. Building Xerces v3.2.2 from source worked, though. I'm now able to build an EXE that starts, but there are still some libraries not found. Once I get this working completely, I will send a complete list of the steps taken.

**Comment by rjplevin** [comment]
See attached.
[Building GCAM on Windows 10.pdf](https://github.com/JGCRI/gcam-doc/files/2852965/Building.GCAM.on.Windows.10.pdf)
